x-pack/plugins/observability_solution/slo/public/embeddable/slo/overview/slo_embeddable_factory.tsx

```diff
@@ -165,10 +165,20 @@ export const getOverviewEmbeddableFactory = ({
               const kqlQuery = groupFilters?.kqlQuery ?? '';
               const groups = groupFilters?.groups ?? [];
               return (
-                <Wrapper>
+                <div
+                  css={css`
+                    width: 100%;
+                    padding: 5px 15px;
+                    overflow: scroll;
+
+                    .euiAccordion__buttonContent {
+                      min-width: 100px;
+                    }
+                  `}
```

Comment:
Could we opt to use EUI token sizes here instead? The translations won't be exact but IMO it's probably good enough?

```suggestion
                  css={({ euiTheme}) => css`
                    width: 100%;
                    padding: ${euiTheme.size.xs} ${euiTheme.size.base};
                    overflow: scroll;

                    .euiAccordion__buttonContent {
                      min-width: ${euiTheme.base * 6}px;
                    }
                  `}
```